### PR TITLE
Add missing openapi_spec.md

### PR DESCRIPTION
**Pull request by tschaffter**
See https://github.com/tschaffter/gh-openapi-docs/issues/3#issuecomment-705233425

### CODE REVIEW

File: test/test-spec/split/tags/openapi_spec.md [hunk 1]
```diff
@@ -0,0 +1 @@
+OpenAPI spec for..
```
Comment (jaeddy) [inline test/test-spec/split/tags/openapi_spec.md]: Here's the original content (not sure where I found it...):

> The goal of The OpenAPI Specification is to define a standard, language-agnostic interface to REST APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection. When properly defined via OpenAPI, a consumer can understand and interact with the remote service with a minimal amount of implementation logic. Similar to what interfaces have done for lower-level programming, OpenAPI removes the guesswork in calling the service.

**Review by jaeddy — CHANGES_REQUESTED** [review]
Thanks for catching this! Do you mind adding in the original text?

File: test/test-spec/split/tags/openapi_spec.md (hunk 1 above)
Comment (tschaffter) [reply test/test-spec/split/tags/openapi_spec.md]: Done

**Review by jaeddy — APPROVED** [review]
Thanks!

**Pull request merged**